lib/css/components/_stacks-buttons.less

```diff
@@ -230,8 +230,7 @@
     //      Creates a gray button style.
     //  ----------------------------------------------------------------------------
     .s-btn__muted {
-        #stacks-internals #load-config();
-        color: @button-muted-color;
+        color: var(--black-500);
```

Comment:
Overall, lgtm. I'd like to consider making these theme variables, but I don't think that's necessary at the moment since we don't have any consumers overriding them.